setup.py

```diff
@@ -16,6 +16,12 @@
       classifiers = [
                      'Development Status :: 4 - Beta',
                      'Programming Language :: Python',
+                     'Programming Language :: Python :: 2.7',
```

Comment:
Can add "Python 2" as well:
```suggestion
                     'Programming Language :: Python :: 2',
                     'Programming Language :: Python :: 2.7',
```